crates/README.md

```diff
@@ -0,0 +1,18 @@
+TODO other crates
```

Comment:
I just want to have this handled before we merge. I can probably get to it tomorrow but anyone is welcome to take it.

Reply:
> I can probably get to it tomorrow

I got deep down a segfault rabbit hole when updating the examples to basic-cli 0.13 :p

Reply:
@lukewilliamboswell  Can you explain what the purpose is of the `roc_app` and `roc_fn` crates and why we don't need those in basic-cli?